scripts/checks.sh

```diff
@@ -199,6 +199,20 @@ function check_dns()
 	log_status "${GOOD}" "${FUNCNAME[0]}" "First DNS server is ${first_server}"
 }
 
+function check_os_rollback()
+{
+	local health_path="/mnt/state/rollback-health-breadcrumb"
+	local altboot_path="/mnt/state/rollback-altboot-breadcrumb"
```

Comment:
`/mnt/state/rollback-altboot-breadcrumb`
`/mnt/state/rollback-health-breadcrumb`

The breadcrumb are left during oldOS by hostapp-update-hooks and cleaned by newOS during a normal HUP.
If new OS fails to clean up, that's a subtly different failure mode.

If rollback actually triggered i.e. HUP failed and the device switched to previous OS, these files are left behind
`/mnt/state/rollback-altboot-triggered`
`/mnt/state/rollback-health-triggered`

Reply:
Good note, thank you! Updated.